doc/reference/configuration/configuration_reference.rst

```diff
@@ -2411,45 +2460,74 @@ The ``process`` defines configuration parameters of the Tarantool process in the
     | Type: boolean
     | Default: false
     | Environment variable: TT_PROCESS_BACKGROUND
-
+    | Dynamic: no
 
 .. _configuration_reference_process_coredump:
 
 .. confval:: process.coredump
 
+    **Since:** :doc:`3.0.0 </release/3.0.0>`.
+
+    Whether to create coredump files.
 
     |
     | Type: boolean
     | Default: false
     | Environment variable: TT_PROCESS_COREDUMP
+    | Dynamic: no
 
 .. _configuration_reference_process_title:
 
 .. confval:: process.title
 
-    Sets the name of the server's process title (what’s shown in the COMMAND column for
-    ``ps -ef`` and ``top -c`` commands).
+    **Since:** :doc:`3.0.0 </release/3.0.0>`.
+
+    Add the given string to the server's process title
+    (what’s shown in the COMMAND column for ``ps -ef`` and ``top -c`` commands).
```

Comment:
> what’s shown in the COMMAND column for ``ps -ef`` and ``top -c`` commands

I'd generalized this description a bit because this look

- There is no COMMAND column on macOS.
- The `-c` flag doesn't work on macOS for the `top` command.

Reply:
My suggestion:

    Add the given string to the server's process title
    (e.g. it's shown in the COMMAND column for the Linux commands
    ``ps -ef`` and ``top -c``).